src/plugins/xp_rpc/xp_rpc.yml

```diff
@@ -0,0 +1,2 @@
+clients:
```

Comment:
This file would need some documentation

Reply:
Yes, I will add documentation for this. It is basically a YAML file that contains a client list of unsecure names and routes. YAML is much easier than INI files, especially with building out a map of client names to routes. Behind the scenes, the xp_rpc plugin will use the client name and route to that location based on the NWNX* function called.

For example, all NWNX* methods require a plugin name, a function name, and a param1. In this plugin, the plugin name is "RPC", the function name is the "client" map key (i.e. "test", "crafting", "health", etc.) and the param1 will be typically used internal to the client to distinguish which function needs to be called. Also could be built as a route or whatever you need out of it.